sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Rules/Hotspots/DoNotHardcodeCredentialsBase.cs

```diff
@@ -58,17 +60,47 @@ public string CredentialWords
             }
         }
```

Comment:
What's \b (backspace character \u0008) check good for?
I dont's see a reason for \S either. SQL password could start with underscore and that would be matched as white space.
And there's missing check for spaces before the delimiter [:=]. 
I think that the regex should look like this:   "(?<password>{0})\s*[:=]"

Reply:
Good point. I've created a new ticket to fix this: #2783 
Note that the `\S` part after the `[:=]` delimiter was to prevent FP in the cases `"Server=localhost; Database=Main; User=Admin; Password= " + loadPassword()`

Reply:
Good point. I've created a new ticket to fix this: #2783 
Note that the `\S` part after the `[:=]` delimiter was to prevent FP in the cases `"Server=localhost; Database=Main; User=Admin; Password= " + loadPassword()`